internal/storage/in_memory.go

```diff
@@ -0,0 +1,77 @@
+package storage
+
+import "github.com/placer14/gof-server/internal/handlers/payloads"
+
+type key_in_memory int
+
+const KeyInMemoryStorage key_in_memory = iota
+
+type InMemoryStorage struct {
+	stringValues map[string]string
+	boolValues   map[string]bool
+	floatValues  map[string]float64
+	intValues    map[string]int64
+}
+
+var _ Storageable = &InMemoryStorage{}
+
+func NewInMemoryStorage() *InMemoryStorage {
+	return &InMemoryStorage{
+		stringValues: make(map[string]string),
+		boolValues:   make(map[string]bool),
+		floatValues:  make(map[string]float64),
+		intValues:    make(map[string]int64),
+	}
+}
+
+func (s *InMemoryStorage) SetString(key, value string) error {
```

Comment:
I'm not sure how well it would work, but maybe generics could simplify this code a bit?